packages/position/src/js/index.ts

```diff
@@ -11,49 +11,57 @@ interface ClipAdjusterProps {
   opts: StyleOptions
 }
 type ClipAdjustment = [number, number]
-type ClipAdjuster = (props: ClipAdjusterProps) => ClipAdjustment
+type ClipAdjuster = (
+  props: ClipAdjusterProps,
+  clipY?: boolean
```

Comment:
How would the JSX use case make use of this if there is not a new prop? Or are you saying there is a new prop I just add it to this arguments? I ask because for Select, MultiSelect, and Typeahead we use the JSX

Reply:
Prop as you have it for jsx is great. When passing to the js functions is what I was referring to.